code/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width , initial-scale=1">
+
+<head>
+  <script src="script-math.js"></script>
+  <script src="script-else.js"></script>
+  <link rel="stylesheet" id="pagestyle" type="text/css" href="theme-default.css">
+  <title>Calculator</title>
+</head>
+
+<body>
+  <h1>Calculator by Roman</h1>
+  <div id="id-container">
+    <form name="nameCalculator" id="id-calculator">
+      <table>
+        <tr>
+          <td colspan="3"><input type="text" name="nameDisplayMain" id="id-displayMain" maxlength="1000" disabled></td>
+          <td><input type="button" name="nameButtonClear" id="id-buttonClear" value="C" onclick="clearDisplay();"></td>
+        </tr>
+        <tr>
+          <td><input type="button" name="nameSeven" class="rcalc-button" value="7" onclick="testZero(nameSeven.value);displayNumber(nameSeven.value);"></td>
```

Comment:
testZero(nameSeven.value);displayNumber(nameSeven.value); ... to many duplicit code ... call just one method and following method call from method directly ... dont forget that this is just html file .. just view ... less logic = more better ... logic you should solve in js file

Reply:
calling just 1 method now, the other one is called from the one we call in html